### PR TITLE
Remove redundent addition of '-n <namespace>'

### PR DESCRIPTION
**Pull request by RazTamir**
Fixes: #1424

Signed-off-by: ratamir <ratamir@redhat.com>

### CODE REVIEW

**Comment by RazTamir** [comment]
Verified locally:

========================================================================================== PASSES ===========================================================================================
------------------------------------------------------------- generated html file: file:///home/ratamir/git/ocs-ci/report.html --------------------------------------------------------------
================================================================================== short test summary info ==================================================================================
PASSED tests/manage/pv_services/test_run_io_multiple_pods.py::TestIOMultiplePods::test_run_io_multiple_pods[CephBlockPool]
PASSED tests/manage/pv_services/test_run_io_multiple_pods.py::TestIOMultiplePods::test_run_io_multiple_pods[CephFileSystem]

File: ocs_ci/ocs/ocp.py [hunk 1]
```diff
@@ -111,7 +112,9 @@ def exec_oc_cmd(
         """
         oc_cmd = "oc "
         kubeconfig = os.getenv('KUBECONFIG')
-        if self.namespace:
+        if all_namespaces and not self.namespace:
+            oc_cmd += "-A"
```
Comment (ebenahar) [inline ocs_ci/ocs/ocp.py]: missing a space, should be "-A "

Comment (RazTamir) [reply ocs_ci/ocs/ocp.py]: line 113:
"oc " includes the space

Comment (petr-balogh) [reply ocs_ci/ocs/ocp.py]: I think that @ebenahar mentioned space at the end not at the start.
As I see that we are adding the command right after and then we will miss the space between. `-A and command`

**Comment by MeridianExplorer** [comment]
Looks good, just needs that space in the end of line 116 - `oc_cmd += "-A "`

**Comment by ebenahar** [comment]
@RazTamir Would like to close old PRs, is this one still relevant?

**Comment by petr-balogh** [comment]
Closing due to inactivity! Feel free to re-open or send the new PR.

**Pull request closed without merging**